src/wrap-store/wrapStore.js

```diff
@@ -47,8 +47,18 @@ export default (store, {
       const action = Object.assign({}, request.payload, {
         _sender: sender
       });
-      dispatchResponder(store.dispatch(action), sendResponse);
-      return true;
+
+      let dispatchResult = null;
+
+      try {
+        dispatchResult = store.dispatch(action);
+      } catch (e) {
+        dispatchResult = Promise.reject(e.message);
+        console.error(e);
+      } finally {
```

Comment:
You can get rid of the `finally` if you'd like. Just a matter of style.

```
let dispatchResult = null;

try {
  dispatchResult = store.dispatch(action);
} catch(e) {
  dispatchResult = Promise.reject(e.message);
  console.error(e);
}

dispatchResponder(store.dispatch(action), sendResponse);
return true;
```

I like this though. A lot. :-)